_docker/drupal/managed-platform/ansible/templates/local/drupal/rhd.settings.php.j2

```diff
@@ -123,6 +123,10 @@ if($config['redhat_developers']['environment'] == 'local') {
     $settings['cache']['bins']['render'] = 'cache.backend.null';
     $settings['cache']['bins']['dynamic_page_cache'] = 'cache.backend.null';
     $settings['cache']['bins']['page'] = 'cache.backend.null';
+
```

Comment:
Add // phpcs:ignoreFile to the top of the rhd.settings.php file

Reply:
Added 👍